common/src/log.rs

```diff
@@ -31,6 +31,13 @@ pub enum Backend {
     Runtime(String),
 }
 
+#[derive(Deserialize)]
+pub enum LogMode {
+    Head,
+    Tail,
+    All,
+}
```

Comment:
I'm not sure we need the `All` variant. Can't we just default to all if no log mode or length is passed? Then in the CLI, we can have a default length set for the logs, and if someone passes `--all` we will send the request without any length or mode.

Reply:
I prefer `All` over having `Option<LogMode>`. My other suggestion allows us to send nothing in the case of `all`.

Reply:
That makes sense, we can go for that!

Reply:
i have to remove Option from both LogMode and len if i understand it correctly?